### PR TITLE
Update bootstrap documentation

### PR DESCRIPTION
**Pull request by DPGrev**
https://github.com/zendesk/maxwell/issues/1361

### CODE REVIEW

**Comment by osheroff** [comment]
cool, but: 

1.  I think you always need to specify `client_id` if you're using SQL. 
2.  The section heading isn't quite right now.

**Comment by DPGrev** [comment]
For me the original command without the client id still works with the default settings:

```
insert into maxwell.bootstrap (database_name, table_name) values ('fooDB', 'barTable');
```

**Comment by osheroff** [comment]
huh!  ok i'll look

File: docs/docs/bootstrapping.md [hunk 1]
```diff
@@ -14,20 +14,30 @@ option                                        | description
...
 
-### Using the maxwell.bootstrap table
+### Starting a table bootstrap
 ***
-Alternatively you can insert a row in the `maxwell.bootstrap` table to trigger a bootstrap.
+You can start a bootstrap using:
 
 ```
-mysql> insert into maxwell.bootstrap (database_name, table_name) values ('fooDB', 'barTable');
+bin/maxwell-bootstrap --config localhost.properties --database fooDB --table barTable --log_level info
```
Comment (osheroff) [inline docs/docs/bootstrapping.md]: I'm not sure why we thought you needed to specify `--config` and `--log_level` in these examples.  As long as we're updating the docs let's simplify this down to just what you need, ie `--database` and `--table`

File: docs/docs/bootstrapping.md [hunk 2]
```diff
@@ -14,20 +14,30 @@ option                                        | description
...
 ```
+
 Optionally, you can include a where clause to replay part of the data.
 
-bin/maxwell-bootstrap --config localhost.properties --database foobar --table test --log_level info
+```
+bin/maxwell-bootstrap --config localhost.properties --database fooDB --table barTable --where "my_date >= '2017-01-07 00:00:00'" --log_level info
```
Comment (osheroff) [inline docs/docs/bootstrapping.md]: here too, `--log_level` and `--config` are just clutter

File: docs/docs/bootstrapping.md (hunk 1 above)
Comment (DPGrev) [reply docs/docs/bootstrapping.md]: Done

File: docs/docs/bootstrapping.md (hunk 2 above)
Comment (DPGrev) [reply docs/docs/bootstrapping.md]: Done

**Pull request merged**